src/routes/auth/invite-user.e2e.test.js

```diff
@@ -0,0 +1,99 @@
+const { faker } = require('@faker-js/faker')
+const { expect } = require('chai')
+const { fixtures } = require('../../../test/utils')
+
+const endpointRoute = '/api/v1/auth/invite-user'
+
+describe('POST /invite-user', () => {   
+	let nonAdminUser
+	let adminUser
+	let nonAdminAccessToken
+	let adminAccessToken
+	let invitedUserInfo
+
+	before(async () => {
+		nonAdminUser = await fixtures.insertUser({
```

Comment:
so aside the before, if we have a variable that is specific to one test case, we can simply move it into it

That is this is sort of used for a global access in multiple test cases. if not you can insert the non admin user and generate the token in just the test cases where it is needed

I feel doing the action exactly at the time it is needed makes it easier to reason about the test flow

Reply:
You are right, thanks a lot